manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -0,0 +1,443 @@
+	configurationInternalAuth := readContentFromFile("pkg/resources/httpd_conf/configuration-internal-auth.conf")
+	configurationExternalAuth := readContentFromFile("pkg/resources/httpd_conf/configuration-external-auth.conf")
+	configurationActiveDirectoryAuth := readContentFromFile("pkg/resources/httpd_conf/configuration-active-directory-auth")
+	configurationSamlAuth := readContentFromFile("pkg/resources/httpd_conf/configuration-saml-auth")
+	configurationOpenidConnectAuth := readContentFromFile("pkg/resources/httpd_conf/configuration-openid-connect-auth")
+	
+	externalAuthLoadModulesConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-load-modules-conf")
+	externalAuthLoginFormConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-login-form-conf")
+	externalAuthApplicationApiConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-application-api-conf")
+	externalAuthLookupUserDetailsConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-lookup-user-details-conf")
+	externalAuthRemoteUserConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-remote-user-conf")
+	externalAuthOpenidConnectRemoteUserConf := readContentFromFile("pkg/resources/httpd_conf/external-auth-openid-connect-remote-user-conf")
+
+
+	data := map[string]string{
+		"application.conf": string(appconf),
+		"authentication.conf": string(authconf),
+		"configuration-internal-auth": string(configurationInternalAuth),
+		"configuration-external-auth": string(configurationExternalAuth),
+		"configuration-active-directory-auth": string(configurationActiveDirectoryAuth),
+		"configuration-saml-auth": string(configurationSamlAuth),
+		"configuration-openid-connect-auth": string(configurationOpenidConnectAuth),
+		"external-auth-load-modules-conf": string(externalAuthLoadModulesConf),
+		"external-auth-login-form-conf": string(externalAuthLoginFormConf),
+		"external-auth-application-api-conf": string(externalAuthApplicationApiConf),
+		"external-auth-lookup-user-details-conf": string(externalAuthLookupUserDetailsConf),
+		"external-auth-remote-user-conf": string(externalAuthRemoteUserConf),
+		"external-auth-openid-connect-remote-user-conf": string(externalAuthOpenidConnectRemoteUserConf),
+	}
+
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "httpd-configs",
+			Namespace: cr.ObjectMeta.Namespace,
+			Labels:    labels,
+		},
+		Data: data,
+	}
+}
+
+func NewHttpdAuthConfigMap(cr *miqv1alpha1.Manageiq) *corev1.ConfigMap {
+	labels := map[string]string{
+		"app": cr.Spec.AppName,   
+	}
+	authconf :=  readContentFromFile("pkg/resources/httpd_conf/authentication.conf")
+	data := map[string]string{
+		"auth-type": "internal",
+		"auth-kerberos-realms": "undefined",
+		"auth-oidc-provider-metadata-url": "undefined",
+    	"auth-oidc-client-id": "undefined",
+    	"auth-oidc-client-secret": "undefined",
+    	"auth-configuration.conf": string(authconf),
```

Comment:
Spacing

Reply:
I think `gofmt` should catch a lot of this right?